Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/MagDurCurves/CurveForm.tsx

```diff
@@ -230,9 +236,9 @@ export default function CurveForm(props: IProps) {
                         zoom={true} pan={false}
                         useMetricFactors={false} XAxisType={'log'}
                         >
-                        <Line highlightHover={false} showPoints={true} lineStyle={'-'}
+                        <Line highlightHover={false} showPoints={false} lineStyle={'-'}
                             color={"#A30000"} data={curve} />
-                        
+                        {curve.map((p,i) => (i == curve.length-1? null : <Circle data={[p[0], p[1]]} color={"#FFFFFF"} radius={8} borderColor={"#A30000"} borderThickness={2} text={String(curve.indexOf(p) + 1)} />))}
```

Comment:
sorry to be super nit-picky but 
```suggestion
                        {curve.map((p,i) => (i == curve.length-1? null : <Circle data={[p[0], p[1]]} color={"#FFFFFF"} radius={8} borderColor={"#A30000"} borderThickness={2} text={String(curve.indexOf(p) + 1)} key={i} />))}
```
Prevents the unique key required warning in the console